x-pack/plugins/session_view/public/components/SessionView/index.tsx

```diff
@@ -89,22 +102,89 @@ export const SessionView = ({ sessionEntityId, height }: SessionViewDeps) => {
     );
   };
 
+  const toggleDetailPanel = () => {
+    setIsDetailMounted(!isDetailMounted);
+    if (!isDetailOpen) setIsDetailOpen(true);
+  };
+
   if (!data.length) {
     return renderNoData();
   }
-
+console.log(selectedProcess)
   return (
     <>
-      <EuiSearchBar query={searchQuery} onChange={onSearch} />
-      <div css={styles.processTree}>
-        <ProcessTree
-          sessionEntityId={sessionEntityId}
-          forward={data}
-          searchQuery={searchQuery}
-          selectedProcess={selectedProcess}
-          onProcessSelected={onProcessSelected}
-        />
-      </div>
+      <EuiFlexGroup>
+        <EuiFlexItem>
+          <EuiSearchBar query={searchQuery} onChange={onSearch} />
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiButton onClick={toggleDetailPanel} iconType="list" fill>
+            Detail Panel
+          </EuiButton>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+      <EuiSplitPanel.Outer direction="row" color="transparent" borderRadius="none" css={styles.outerPanel}>
+        <EuiSplitPanel.Inner paddingSize="none" css={styles.treePanel}>
+          <div css={styles.processTree}>
+            <ProcessTree
+              sessionEntityId={sessionEntityId}
+              forward={data}
+              searchQuery={searchQuery}
+              selectedProcess={selectedProcess}
+              onProcessSelected={onProcessSelected}
+            />
+          </div>
+        </EuiSplitPanel.Inner>
+        {isDetailOpen && (
+          <EuiSplitPanel.Inner
+            paddingSize="s"
+            color="plain"
+            css={isDetailMounted ? styles.detailPanelIn : styles.detailPanelOut}
+            onAnimationEnd={() => {
+              if (!isDetailMounted) setIsDetailOpen(false);
+            }}
+          >
+            {selectedProcess && (
+              <>
+                <EuiTitle size="xs"><span>Command Detail</span></EuiTitle>
```

Comment:
i'd say localize, but i imagine this UX will change once design gets it's hands on it.. There are some things to consider though. 1 process can have many events. e.g fork, exec, exec, exit, output, alert etc... so rendering all of these in details could be a lot of repeated info. I have a getDetails() funciton in process which returns the most recent fork or exec (though it should probably return exit if exists, so we can show info like exit_code.. but yea. having access to all the lifecycle events is useful as well...

Reply:
Yea I agree. maybe we can just show all the events for now and make changes after we get some feedback from UX